api-docs/api-reference/methods/roles/get-list-effective-roles-assigned-to-a-user-v2.0.rst

```diff
@@ -0,0 +1,400 @@
+.. _get-list-effective-roles-assigned-to-a-user-v2.0:
+
+List effective roles assigned to a user
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code::
+
+    GET /v2.0/users/{userId}/RAX-AUTH/roles
+
+This operation returns the list of all roles that are assigned to the user with
+the specified user Id. If you don't know the user ID, use the
+:ref:`Get user by Id <get-user-by-id-v2.0>` operation to find it.
```

Comment:
I think you mean by name?  If you don't know the id, this call would be useless I think

Reply:
Yep, that would make perfect sense. I'll make the change.